[Snyk] Fix for 7 vulnerabilities

caniszczyk opened the pull request:
![snyk-top-banner](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### Snyk has created this PR to fix 7 vulnerabilities in the pip dependencies of this project.

#### Snyk changed the following file(s):

- `builders/testdata/python/custom_entrypoint/requirements.txt`






---

> [!IMPORTANT]
>
> - Check the changes in this PR to ensure they won't cause issues with your project.
> - Max score is 1000. Note that the real score may have changed since the PR was raised.
> - This PR was automatically created by Snyk using the credentials of a real user.
> - Some vulnerabilities couldn't be fully fixed and so Snyk will still find them when the project is tested again. This may be because the vulnerability existed within more than one direct dependency, but not all of the affected dependencies could be upgraded.

---

**Note:** _You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs._

For more information: <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiI1ODNkNGVjZS01MDNhLTRiYWYtOGQzNS1kYzEwYmNhMjMwN2YiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6IjU4M2Q0ZWNlLTUwM2EtNGJhZi04ZDM1LWRjMTBiY2EyMzA3ZiJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.snyk.io/org/openfunction/project/996ebcaf-0e6d-4796-a520-fc1e8ed8d778?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)
📜 [Customise PR templates](https://docs.snyk.io/scan-using-snyk/pull-requests/snyk-fix-pull-or-merge-requests/customize-pr-templates?utm_source=github&utm_content=fix-pr-template)
🛠 [Adjust project settings](https://app.snyk.io/org/openfunction/project/996ebcaf-0e6d-4796-a520-fc1e8ed8d778?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)
📚 [Read about Snyk's upgrade logic](https://docs.snyk.io/scan-with-snyk/snyk-open-source/manage-vulnerabilities/upgrade-package-versions-to-fix-vulnerabilities?utm_source=github&utm_content=fix-pr-template)

---

**Learn how to fix vulnerabilities with free interactive lessons:**

🦉 [Cross-site Scripting (XSS)](https://learn.snyk.io/lesson/xss/?loc&#x3D;fix-pr)
🦉 [Access Restriction Bypass](https://learn.snyk.io/lesson/broken-access-control/?loc&#x3D;fix-pr)
🦉 [Denial of Service (DoS)](https://learn.snyk.io/lesson/no-rate-limiting/?loc&#x3D;fix-pr)

[//]: # 'snyk:metadata:{"customTemplate":{"variablesUsed":[],"fieldsUsed":[]},"dependencies":[{"name":"jinja2","from":"2.11.3","to":"3.1.6"},{"name":"werkzeug","from":"1.0.1","to":"2.2.3"}],"env":"prod","issuesToFix":["SNYK-PYTHON-JINJA2-6150717","SNYK-PYTHON-JINJA2-6809379","SNYK-PYTHON-JINJA2-8548181","SNYK-PYTHON-JINJA2-8548987","SNYK-PYTHON-JINJA2-9292516","SNYK-PYTHON-WERKZEUG-3319935","SNYK-PYTHON-WERKZEUG-3319936"],"prId":"583d4ece-503a-4baf-8d35-dc10bca2307f","prPublicId":"583d4ece-503a-4baf-8d35-dc10bca2307f","packageManager":"pip","priorityScoreList":[484,484,484,484,484,344,589],"projectPublicId":"996ebcaf-0e6d-4796-a520-fc1e8ed8d778","projectUrl":"https://app.snyk.io/org/openfunction/project/996ebcaf-0e6d-4796-a520-fc1e8ed8d778?utm_source=github&utm_medium=referral&page=fix-pr","prType":"fix","templateFieldSources":{"branchName":"default","commitMessage":"default","description":"default","title":"default"},"templateVariants":["priorityScore"],"type":"auto","upgrade":[],"vulns":["SNYK-PYTHON-JINJA2-6150717","SNYK-PYTHON-JINJA2-6809379","SNYK-PYTHON-JINJA2-8548181","SNYK-PYTHON-JINJA2-8548987","SNYK-PYTHON-JINJA2-9292516","SNYK-PYTHON-WERKZEUG-3319935","SNYK-PYTHON-WERKZEUG-3319936"],"patch":[],"isBreakingChange":false,"remediationStrategy":"vuln"}'

snyk-bot committed:
fix: builders/testdata/python/custom_entrypoint/requirements.txt to reduce vulnerabilities


The following vulnerabilities are fixed by pinning transitive dependencies:
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6150717
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6809379
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548181
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548987
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-9292516
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319935
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319936

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/OpenFunction/gcp-builder?pullRequest=76) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/OpenFunction/gcp-builder?pullRequest=76) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/OpenFunction/gcp-builder?pullRequest=76) it.</sub>

snyk-bot committed:
fix: builders/testdata/python/custom_entrypoint/requirements.txt to reduce vulnerabilities


The following vulnerabilities are fixed by pinning transitive dependencies:
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6150717
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6809379
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548181
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548987
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-9292516
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319935
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319936

snyk-bot committed:
fix: builders/testdata/python/custom_entrypoint/requirements.txt to reduce vulnerabilities


The following vulnerabilities are fixed by pinning transitive dependencies:
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6150717
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6809379
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548181
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548987
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-9292516
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319935
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319936

snyk-bot committed:
fix: builders/testdata/python/custom_entrypoint/requirements.txt to reduce vulnerabilities


The following vulnerabilities are fixed by pinning transitive dependencies:
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6150717
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6809379
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548181
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548987
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-9292516
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319935
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319936

snyk-bot committed:
fix: builders/testdata/python/custom_entrypoint/requirements.txt to reduce vulnerabilities


The following vulnerabilities are fixed by pinning transitive dependencies:
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6150717
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-6809379
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548181
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-8548987
- https://snyk.io/vuln/SNYK-PYTHON-JINJA2-9292516
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319935
- https://snyk.io/vuln/SNYK-PYTHON-WERKZEUG-3319936

benjaminhuo commented:
## ✅ This PR has been automatically closed

The security issues addressed by this pull request are no longer present in the latest project scan. All vulnerabilities this PR was created to fix have been resolved through other means (e.g., dependency updates, direct fixes, or changes in vulnerability data).

### Resolved Issues
- `SNYK-PYTHON-JINJA2-6150717`
- `SNYK-PYTHON-JINJA2-6809379`
- `SNYK-PYTHON-JINJA2-8548181`
- `SNYK-PYTHON-JINJA2-8548987`
- `SNYK-PYTHON-JINJA2-9292516`
- `SNYK-PYTHON-WERKZEUG-3319935`
- `SNYK-PYTHON-WERKZEUG-3319936`

### What should I do?
No action is required. If you believe this PR was closed in error, you can reopen it and contact Snyk support.

---
*This action was performed automatically by Snyk.*
